vector/src/main/java/im/vector/app/features/poll/create/CreatePollViewState.kt

```diff
@@ -14,19 +14,21 @@
  * limitations under the License.
  */
 
-package im.vector.app.features.createpoll
+package im.vector.app.features.poll.create
 
 import com.airbnb.mvrx.MavericksState
 
 data class CreatePollViewState(
         val roomId: String,
         val question: String = "",
         val options: List<String> = emptyList(),
-        val canCreatePoll: Boolean = false
+        val canCreatePoll: Boolean = false,
+        val canAddMoreOptions: Boolean = true
 ) : MavericksState {
 
     constructor(args: CreatePollArgs) : this(
-            roomId = args.roomId
+            roomId = args.roomId,
+            options = List(args.minOptionsCount) { "" }
```

Comment:
This is not the best place to init `options`. I would have done it on line 24 instead

Reply:
Done.